subfun/BIDS_FFX.m

```diff
@@ -94,67 +94,67 @@ function BIDS_FFX(action, degreeOfSmoothing, opt)
                 matlabbatch{1}.spm.stats.fmri_spec.dir = {ffxDir};
                 % FMRI DESIGN
                 fprintf(1,'BUILDING JOB : FMRI design\n')
-                
+
                 sesCounter = 1;
-                
-                
+
+
                 %%
                 % identify sessions for this subject
-                [sessions, numSessions] = getSessions(BIDS, subNumber, opt);
-                
+                [sessions, numSessions] = getInfo(BIDS, subNumber,'Sessions',opt);
```

Comment:
I think we should then change the order of the arguments in the call of the function too, no?

So for example:

`[sessions, numSessions] = getInfo(BIDS,  subNumber, opt, 'Sessions');`

instead of 

`[sessions, numSessions] = getInfo(BIDS, subNumber,'Sessions',opt);`

Reply:
True, I changed it. 
do I have to create a new pull request or modifying the files in the branch will update the files in the previous pull request?

Reply:
yes modifying the lines in the file and commiting + pushing will update the pull request.

you can actually see it above even here because my comment has the `outdated` tag.